### DISCUSSION TITLE
docs source code

### DISCUSSION THREAD

**Discussion by o-az** [x-cmd/x-cmd#101] [Q&A]
is the source code for https://www.x-cmd.com open source?

**Comment by edwinjhlee** [comment]
No. The source code of website is not opensourced.

**Reply by o-az** [reply]
@edwinjhlee can I ask you what tool you're using for the code blocks with ability to fast forward and rewind?

![image](https://github.com/user-attachments/assets/05f416f7-6d51-4031-9c1c-44ef7bb853b6)

**Reply by edwinjhlee** [reply]
asciinema player.

**Reply by edwinjhlee** [reply]
https://asciinema.org/
